bin/postinstall

```diff
@@ -1,3 +1,48 @@
 #!/usr/bin/env bash
 
-asdf reshim nodejs "$ASDF_INSTALL_VERSION"
+set -e
+
+declare LOCK_FILE="/tmp/asdf-nodejs-postinstall.lock.2ac8ccc4"
+declare LAST_CALLED_FILE="/tmp/asdf-nodejs-postinstall.last_called.dd05fa02"
```

Comment:
These paths should be dynamic. We should probably use `mktemp` for these files.

Reply:
We need one common lock file and one common last_called file. We can check if `/tmp/asdf-nodejs-postinstall.last_called.*` exists and use the existing one if it does, generating it otherwise.
We would have to run it in the foreground thus making it a little slower.
Actually, we can even safely drop the `.2ac8ccc4` and `.dd05fa02` suffixes, it's just me being a little paranoic.

Reply:
Yes you can get rid of those hashes if there is only going to be at most one file at a time.